registry/handlers/manifests.go

```diff
@@ -485,6 +485,11 @@ func (imh *manifestHandler) applyResourcePolicy(manifest distribution.Manifest)
 func (imh *manifestHandler) DeleteManifest(w http.ResponseWriter, r *http.Request) {
 	dcontext.GetLogger(imh).Debug("DeleteImageManifest")
 
+	if imh.Tag != "" {
```

Comment:
Do I understand correctly that `imh.Tag` and `imh.Digest` are mutually exclusive? Just curious if we should also check that `imh.Digest == ""`.

Reply:
Good question. Yes, one or the other gets populated when the `manifestHandler` is instantiated [here](https://github.com/digitalocean/docker-distribution/blob/master/registry/handlers/manifests.go#L48).